src/makeCalculator.js

```diff
@@ -38,6 +38,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
Make it like a pure function.
Let this method return you a pure result, and save it in your operate method.
For example

```
add(num1, num2) {
	return num1 + num2;
}